### PR TITLE
feat(dashboard-date-range-modal): create new custom date range modal & renewal toolset

### PR DESCRIPTION
**Pull request by piggggggggy**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [x] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
- `CustomDateRangeModal` is changed. So, I create New `DashboardCustomDateRangeModal.vue`.
- Currency and date-range-badge is removed.
- Remove default date range in `dashboard-detail-info` store. default value is `undefined`, that is shown as `Current Month`.

- Changed Toolset
image

- New Modal
image

Please, see order by commits.

**_Dropdown's new design will be applied in next PR_**

### CODE REVIEW

File: apps/web/src/services/dashboards/shared/DashboardCustomDateRangeModal.vue [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    selectedDateRange: () => ({}),
+});
+
+const emit = defineEmits<{(event: 'update:visible', visible: boolean): void;
+    (event: 'confirm', period: Period): void;
+}>();
+
+const state = reactive({
+    proxyVisible: useProxyValue('visible', props, emit),
+    dateType: 'yearToMonth',
```
Comment (yuda110) [inline apps/web/src/services/dashboards/shared/DashboardCustomDateRangeModal.vue:25]: nitpick: !!

```
import { DATA_TYPE } from '@spaceone/design-system/src/inputs/datetime-picker/type';
DATA_TYPE.yearToMonth
```

**Review by yuda110 — COMMENTED** [review]
I left questions in Slack!

File: apps/web/src/services/dashboards/shared/DashboardCustomDateRangeModal.vue [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+                                   :selected-dates="state.selectedDates"
+                                   :invalid="state.invalid"
+                                   :min-date="state.dateLimitSetting.minDate"
+                                   :max-date="state.dateLimitSetting.maxDate"
+                                   @update:selected-dates="handleUpdateSelectedDates"
+                />
+            </p-field-group>
+        </template>
+        <template #confirm-button>
+            {{ $t('Apply') }}
```
Comment (sulmoJ) [inline apps/web/src/services/dashboards/shared/DashboardCustomDateRangeModal.vue:100]: Translation!?

**Review by sulmoJ — APPROVED** [review]
LGTM!~

**Pull request merged**